mysql-test/suite/innodb/t/bug84563.test

```diff
@@ -0,0 +1,37 @@
+--source include/have_innodb.inc
```

Comment:
This is a feature, not bugfix, consider naming as percona_print_lock_wait_timeout_trx

Reply:
ok